kustomize/README.md

```diff
@@ -8,6 +8,8 @@ Kustomize is a Kubernetes configuration management tool that allows users to cus
 
 ## Prerequisites
 
+You need to have the kustomize binary installed. Instructions can be found here: https://kubectl.docs.kubernetes.io/installation/
+
```

Comment:
100% I think that's fine. I was just following the pattern from the Spanner integration, and then because I hit the problem, I added the note as clarity for folks that might also hit it.